app/services/profile_fields/field_definition.rb

```diff
@@ -8,8 +8,8 @@ def call
         new.add_fields
       end
 
-      def group(group = nil)
-        @group = group
+      def group(description = nil)
```

Comment:
I'm glad I made this little DSL, so I didn't have to go and make too many changes elsewhere.